### ISSUE TITLE
Unstable training of DPRNN-TasNet

### ISSUE THREAD

**Issue by tky823** [tky823/DNN-based_source_separation#43]
The network wouldn't be trained when
- Layer normalization is applied before the bottleneck convolution.
- Batch size / GPU is small (e.g. 1).
  - This phenomenon is observed in Conv-TasNet.

No bottleneck convolution seems to be better.

**Comment by tky823** [comment]
Related to #23

**Comment by tky823** [comment]
The bias of bottleneck convolution?

**Comment by tky823** [comment]
Possible causes ...
- Layer normalization
- Bottleneck convolution
  - The bias of bottleneck convolution
- Batch size / GPU is small
  ... This phenomenon is observed in Conv-TasNet.
- The trainer
  ... Gradient clipping?
